build.gradle

```diff
@@ -453,6 +464,35 @@ tasks.register("buildReleaseArtifacts").configure {
     .findAll { it != null }
 }
 
+/**
+ * This tasks allows third party open source user to build all artifacts
+ * that are build from public available sources
+ * We used to propose to run `./gradlew assemble` but this leads to:
+ * - building artifacts that are not required for a user (e.g. internally used test fixtures)
+ * - failure when building docker images with non public base images (e.g. wolfi)
+ * */
+tasks.register("assemblePublicArtifacts").configure {
```

Comment:
I don't think there's value in such a task. There's no reason why anyone would want to build all these artifacts in one go and it just adds confusion. Let's omit this.